### ISSUE TITLE
A version bump to a runtimeOnly dependency does not mark module as affected

### ISSUE THREAD

**Issue by roded** [dropbox/AffectedModuleDetector#207]
I have a module with a `runtimeOnly` dependency. When bumping that dependency's version, AMD does not mark the module as affected. However, if the same dependency is changed to the `implementation` configuration AMD picks it up.

Is this intended?

**Comment by roded** [comment]
I think that the issue is that this plugin does not play nice with Gradle version catalog toml files which do not reside in the project.

**Comment by edenman** [comment]
fwiw I got around this by doing `pathsAffectingAllModules = setOf("gradle/libs.versions.toml")`.  Annoying, but works great

**Comment by joshafeinberg** [comment]
Yes, that is probably the proper fix to listen to the toml file. Happy to accept a PR if that doesn't resolve your issue
